### DISCUSSION TITLE
Counter returns zero in typescript

### DISCUSSION THREAD

**Discussion by rogue-kitten** [MasterKale/SimpleWebAuthn#445] [Troubleshooting]
- While registering a device, the counter always returns zero in typescript. I am not sure if this is the correct behaviour or not. The implementation is exactly the same that has been showcased as an example here in the repo

**Comment by MasterKale** [comment]
The counter my methods give you back is the counter reported by the authenticator. Often platform authenticators will only ever give back 0 so in my experience what you're seeing is expected.

If you have a security key handy that should return actual incrementing numbers to validate this.
